.circleci/check_csv.py

```diff
@@ -7,17 +7,16 @@
 
 def check_csv(filename):
     """
-    Basic accuracy checking. If a model fails, the speedup will be 0.
+    Basic accuracy checking.
     """
 
-    actual = pd.read_csv(filename)
+    df = pd.read_csv(filename)
 
     failed = []
-    for model_name in actual["name"]:
-        speedup = float(actual.loc[actual["name"] == model_name]["speedup"])
-        status = "PASS"
-        if speedup == 0:
-            status = "FAIL"
+    for _, row in df.iterrows():
+        model_name = row["name"]
+        status = row["accuracy"]
+        if status != "pass":
```

Comment:
With this change, our CI coverage will drop significantly if we skip all the failed-accuracy-check benchmarks. With the uncertainty on when we can fix those errors, this worries me. Can we distinguish accuracy check failures and other failures, and only check for other failures here?

Reply:
This worries me as well. One way is to just fallback to skip_accuracy_check for now, and get this PR in. This will ensure that non-accuracy failures will get tested through CI.

And then prioritize the accuracy failures. What do you think?